extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -249,8 +249,8 @@ export class AmpA4A extends AMP.BaseElement {
     /** @private {?AMP.AmpAdXOriginIframeHandler} */
     this.xOriginIframeHandler_ = null;
 
-    /** @private {boolean} whether creative has been verified as AMP */
-    this.isVerifiedAmpCreative_ = false;
+    /** @type {boolean} whether creative has been verified as AMP */
+    this.isVerifiedAmpCreative = false;
```

Comment:
You only need to know if this is true so rather than allowing sub-classes to potentially modify, please make an accessor (isVerifiedCreative() { return this.isVerifiedCreative_; })

Reply:
Done.